Method: <org.apache.commons.math.exception.util.ExceptionContext: java.lang.String getLocalizedMessage()>, Callee: {throwable={max=100, context=callee,}, msgPatterns=[ILLEGAL_STATE, MAX_COUNT_EXCEEDED, EVALUATIONS], msgArguments=[[Ljava.lang.Object;@4f57b373, [Ljava.lang.Object;@2452433b, [Ljava.lang.Object;@5a1d03c3], context={},}, Params: [], Related test: test601(org.apache.commons.math.analysis.solvers.Math40AB_4_Test)
Method: <org.apache.commons.math.exception.MathIllegalStateException: org.apache.commons.math.exception.util.ExceptionContext getContext()>, Callee: {context={throwable=callee, msgPatterns=[ILLEGAL_STATE], msgArguments=[[Ljava.lang.Object;@56277514], context={},}, max=null,}, Params: [], Related test: test600(org.apache.commons.math.analysis.solvers.Math40AB_3_Test)
Method: <org.apache.commons.math.exception.MaxCountExceededException: java.lang.Number getMax()>, Callee: {context={throwable=callee, msgPatterns=[ILLEGAL_STATE, MAX_COUNT_EXCEEDED], msgArguments=[[Ljava.lang.Object;@56277514, [Ljava.lang.Object;@31abf6a8], context={},}, max=100,}, Params: [], Related test: test600(org.apache.commons.math.analysis.solvers.Math40AB_3_Test)
Method: <org.apache.commons.math.exception.MathIllegalStateException: org.apache.commons.math.exception.util.ExceptionContext getContext()>, Callee: {max=100, context={throwable=callee, msgPatterns=[ILLEGAL_STATE, MAX_COUNT_EXCEEDED], msgArguments=[[Ljava.lang.Object;@4f57b373, [Ljava.lang.Object;@2452433b], context={},},}, Params: [], Related test: test601(org.apache.commons.math.analysis.solvers.Math40AB_4_Test)
Method: <org.apache.commons.math.exception.util.ExceptionContext: void addMessage(org.apache.commons.math.exception.util.Localizable,java.lang.Object[])>, Callee: {throwable={max=null, context=callee,}, msgPatterns=[ILLEGAL_STATE], msgArguments=[[Ljava.lang.Object;@4f57b373], context={},}, Params: [{sourceFormat=maximal count ({0}) exceeded,},[100, []]], Related test: test601(org.apache.commons.math.analysis.solvers.Math40AB_4_Test)
Method: <org.apache.commons.math.exception.MathIllegalStateException: java.lang.String getLocalizedMessage()>, Callee: {max=100, context={throwable=callee, msgPatterns=[ILLEGAL_STATE, MAX_COUNT_EXCEEDED, EVALUATIONS], msgArguments=[[Ljava.lang.Object;@4f57b373, [Ljava.lang.Object;@2452433b, [Ljava.lang.Object;@5a1d03c3], context={},},}, Params: [], Related test: test601(org.apache.commons.math.analysis.solvers.Math40AB_4_Test)
Method: <org.apache.commons.math.exception.util.ExceptionContext: void addMessage(org.apache.commons.math.exception.util.Localizable,java.lang.Object[])>, Callee: {throwable={context=callee, max=null,}, msgPatterns=[ILLEGAL_STATE], msgArguments=[[Ljava.lang.Object;@56277514], context={},}, Params: [{sourceFormat=maximal count ({0}) exceeded,},[100, []]], Related test: test600(org.apache.commons.math.analysis.solvers.Math40AB_3_Test)
Method: <org.apache.commons.math.exception.util.ExceptionContext: java.lang.String buildMessage(java.util.Locale,java.lang.String)>, Callee: {throwable={max=100, context=callee,}, msgPatterns=[ILLEGAL_STATE, MAX_COUNT_EXCEEDED, EVALUATIONS], msgArguments=[[Ljava.lang.Object;@4f57b373, [Ljava.lang.Object;@2452433b, [Ljava.lang.Object;@5a1d03c3], context={},}, Params: [zh_TW,: ], Related test: test601(org.apache.commons.math.analysis.solvers.Math40AB_4_Test)
Method: <org.apache.commons.math.exception.util.ExceptionContext: java.lang.String getMessage(java.util.Locale)>, Callee: {throwable={max=100, context=callee,}, msgPatterns=[ILLEGAL_STATE, MAX_COUNT_EXCEEDED, EVALUATIONS], msgArguments=[[Ljava.lang.Object;@4f57b373, [Ljava.lang.Object;@2452433b, [Ljava.lang.Object;@5a1d03c3], context={},}, Params: [zh_TW], Related test: test601(org.apache.commons.math.analysis.solvers.Math40AB_4_Test)
Method: <org.apache.commons.math.exception.MathIllegalStateException: org.apache.commons.math.exception.util.ExceptionContext getContext()>, Callee: {max=null, context={throwable=callee, msgPatterns=[ILLEGAL_STATE], msgArguments=[[Ljava.lang.Object;@4f57b373], context={},},}, Params: [], Related test: test601(org.apache.commons.math.analysis.solvers.Math40AB_4_Test)
Method: <org.apache.commons.math.exception.util.ExceptionContext: void addMessage(org.apache.commons.math.exception.util.Localizable,java.lang.Object[])>, Callee: {throwable={max=100, context=callee,}, msgPatterns=[ILLEGAL_STATE, MAX_COUNT_EXCEEDED], msgArguments=[[Ljava.lang.Object;@4f57b373, [Ljava.lang.Object;@2452433b], context={},}, Params: [{sourceFormat=evaluations,},[]], Related test: test601(org.apache.commons.math.analysis.solvers.Math40AB_4_Test)
